sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 7.1.0-beta.2 (2021-04-07)
+
+- [Bug Fix] `expiresAtUtc` is `Invalid Date` in the received message when the ttl is not defined. Has been fixed in [#13543](https://github.com/Azure/azure-sdk-for-js/pull/13543)
```

Comment:
Just a note, I have another PR where I'm just removing the date for .beta.1 since we never released it. So your changes would actually go there.

Reply:
Oops, forgot!